lib/foreman_maintain/concerns/os_facts.rb

```diff
@@ -63,7 +71,23 @@ def el8?
       end
 
       def el_major_version
-        return os_version_id.to_i if el?
+        os_version_id.to_i if el?
+      end
+
+      def deb_major_version
+        os_version_id.to_i if debian?
+      end
+
+      def ubuntu_major_version
+        os_version_id.to_i if ubuntu?
```

Comment:
I don't think this is correct: in Ubuntu the full version numbers are strings and should be treated as such. Otherwise you can't distinguish the releases:
```
[1] pry(main)> '20.04'.to_i
=> 20
[2] pry(main)> '20.10'.to_i
=> 20
```

Reply:
While technically true, we really only support Ubuntu LTS versions, so differentiating 18 vs 20 vs 22 is perfectly sufficient. Whether it's valid to call that the *major* number might be a different q ;)

Reply:
I vote for having separate pr to move the Gem::Version methods separate and then use it for versions and comparisons all over the project.